src/sos/workflow_engines.py

```diff
@@ -182,17 +188,8 @@ def purge_workflows(self,
                         tags=None,
                         verbosity=2):
         try:
-            return self.agent.check_output(
-                "{} purge {} {} {} {} {} -v {}".format(
-                    self.agent.config.get("sos", "sos"),
-                    " ".join(workflows),
-                    "--all" if purge_all else "",
-                    f"--age {age}" if age is not None else "",
-                    f'--status {" ".join(status)}'
-                    if status is not None else "",
-                    f'--tags {" ".join(tags)}' if tags is not None else "",
-                    verbosity,
-                ))
+            return self.agent.check_output(check_output2
```

Comment:
You cannot move the definition of the string before the parameters are used.

Reply:
I recently submitted a commit where check_output2 is under 'try:' in purge_workflows and between return.self.agent.check_output().